xml/System.Windows.Forms/ToolStripItem.xml

```diff
@@ -1858,9 +1858,12 @@
         <param name="dragImage">To be added.</param>
         <param name="cursorOffset">To be added.</param>
         <param name="useDefaultDragImage">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Begins a drag operation. The <paramref name="allowedEffects" /> determine which drag operations can occur. If the drag operation needs to interop with applications in another process, <paramref name="data" /> should either be a base managed class (<see cref="T:System.String" />, <see cref="T:System.Drawing.Bitmap" />, or <see cref="T:System.Drawing.Imaging.Metafile" />) or some <see cref="T:System.Object" /> that implements <see cref="T:System.Runtime.Serialization.ISerializable" />. <paramref name="data" /> can also be any <see cref="T:System.Object" /> that implements <see cref="T:System.Windows.Forms.IDataObject" />. <paramref name="dragImage" /> is the bitmap that will be displayed during the  drag operation and <paramref name="cursorOffset" /> specifies the location of the cursor within <paramref name="dragImage" />, which is an offset from the upper-left corner. Specify <see langword="true" /> for <paramref name="useDefaultDragImage" /> to use a layered window drag image with a size of 96x96; otherwise <see langword="false" />. Note the outer edges of <paramref name="dragImage" /> are blended out if the image width or height exceeds 300 pixels.</summary>
+        <returns>A value from the <see cref="T:System.Windows.Forms.DragDropEffects" /> enumeration that represents the final effect that was performed during the drag-and-drop operation.</returns>
+        <remarks>
+          <para> Because <see cref="M:System.Windows.Forms.ToolStripItem.DoDragDrop(System.Object,System.Windows.Forms.DragDropEffects,System.Drawing.Bitmap,System.Drawing.Point,System.Boolean)" /> always performs the RGB multiplication step in calculating the alpha value, you should always pass a <see cref="T:System.Drawing.Bitmap" /> without premultiplied alpha blending. Note that no error will result from passing a <see cref="T:System.Drawing.Bitmap" /> with premultiplied alpha blending, but this method will multiply it again, doubling the resulting alpha value.
```

Comment:
```suggestion
          <para> Because <see cref="M:System.Windows.Forms.ToolStripItem.DoDragDrop(System.Object,System.Windows.Forms.DragDropEffects,System.Drawing.Bitmap,System.Drawing.Point,System.Boolean)" /> always performs the RGB multiplication step in calculating the alpha value, you should always pass a <see cref="T:System.Drawing.Bitmap" /> without premultiplied alpha blending. No error will result from passing a <see cref="T:System.Drawing.Bitmap" /> with premultiplied alpha blending, but this method will multiply it again, doubling the resulting alpha value.
```

Reply:
👍